tests/cases/lookup/renamed_elements.slint

```diff
@@ -0,0 +1,27 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+
+PopupWindow := Rectangle {}
+Rectangle := Image {}
+TabWidget := Text {}
+VerticalLayout := PopupWindow {}
```

Comment:
This means the `VerticalLayout` will be a "real" `Rectangle`, right?

Reply:
in that test, yes